mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/service/impl/IdentityServiceImpl.java

```diff
@@ -54,4 +64,27 @@ public IdentityData getIdentity(String individualId) throws MockIdentityExceptio
 		return identityData;
 	}
 
+	@Override
+	public void addVerifiedClaim(VerifiedClaimRequestDto verifiedClaimRequestDto) throws MockIdentityException {
+		VerifiedClaim verifiedClaim =null;
+		getIdentity(verifiedClaimRequestDto.getIndividualId());
```

Comment:
Not just checking the identity, we should also check if the value is available for the requesting claim

Reply:
added check for claim avialability